src/components/SingleBook.jsx

```diff
@@ -7,37 +7,38 @@ const SingleBook = ({
 }) => {
   const dispatch = useDispatch();
   return (
-    <li className=" flex justify-between">
-      <div>
-        <p>{category}</p>
-        <h2>{title}</h2>
-        <p>{author}</p>
-        <ul className=" flex gap-1">
-          <li>
+    <li className="flex p-5 md:pr-20 gap-4 md:gap-0 flex-col md:flex-row justify-between md:items-center shadow bg-white md:p-10 border border-slate-200 rounded-md">
+      <article>
+        <p className=" text-slate-600 text-sm font-bold capitalize ">{category}</p>
+        <h2 className=" font-bold text-3xl">{title}</h2>
+        <p className=" text-sm text-[rgb(67,134,191)]">{author}</p>
+        <ul className=" flex gap-2 mt-4 text-[rgb(67,134,191)]">
+          <li className="pr-3">
             <button type="button">Comments</button>
           </li>
-          <li>
+          <li className=" border-l px-3">
             <button type="button" onClick={() => dispatch(deleteBookFromServer({ item_id: id }))}>Remove</button>
           </li>
-          <li>
+          <li className=" border-l px-3">
             <button type="button">Edit</button>
           </li>
         </ul>
-      </div>
-      <div className="flex gap-10">
-        <div className="flex gap-2">
+      </article>
+      <article className="flex justify-between items-center gap-10">
+        <div className="flex flex-col md:flex-row md:items-center gap-1">
           <div className="progress" />
           <div>
-            <p>64%</p>
+            <p className=" font-medium text-2xl">64%</p>
             <span>completed</span>
           </div>
         </div>
+        <div className=" w-[60px] h-[2px] bg-gray-200 rotate-90" />
         <div>
-          <h3>CURRENT CHAPTER</h3>
+          <h3 className=" text-gray-400">CURRENT CHAPTER</h3>
           <p>Chapter 17</p>
-          <button type="button">UPDATE PROGRESS</button>
+          <button type="button" className="bg-[rgb(2,144,255)] px-2 py-1 mt-4 text-white rounded">UPDATE PROGRESS</button>
```

Comment:
- [ ] I'd also suggest adding some padding to the left and right of the "UPDATE PROGRESS" button as your design doesn't have enough space as shown in the original design below
- Original design:
  ![image](https://github.com/Akorede3133/bookstore/assets/104968236/bad9c001-37fe-4fb1-a778-5f5b304aed35)
- Your design:
   ![image](https://github.com/Akorede3133/bookstore/assets/104968236/3609167e-24f1-4700-857f-6431707e1d64)